bundles/Eiffel/seed-data/ei_allevents/subscription.json

```diff
@@ -17,7 +17,7 @@
         {
             "conditions" : [
                 {
-                    "jmespath" : "meta.type=='EiffelArtifactCreatedEvent' && data.gav.groupId=='com.othercompany.library'"
+                    "jmespath" : "meta.type=='EiffelArtifactCreatedEvent' && identity=='pkg:maven/com.othercompany.library/required@required'"
```

Comment:
Shouldn't it be data.identity?

Reply:
yepp, thats true, it should be data.identity according to documentation:
https://github.com/eiffel-community/eiffel/blob/master/eiffel-vocabulary/EiffelArtifactCreatedEvent.md

Will update.

Reply:
Fixed.